full-service/src/json_rpc/v2/api/request.rs

```diff
@@ -53,6 +53,16 @@ pub enum JsonCommandRequest {
         max_spendable_value: Option<String>,
         comment: Option<String>,
     },
+    build_burn_transaction {
+        account_id: String,
+        amount: Amount,
+        redemption_memo: Option<String>,
```

Comment:
Suggest to rename to `redemption_memo_hex` to indicate its a hex string not a user-readable string.